### PR TITLE
Subscribe to Entity new incidents

### PR DESCRIPTION
**Pull request by pdcp1**
Part of issue #1299

Screen Shot 2022-11-07 at 17 49 24

- If the user is subscribed to the Entity:
image

- if the user is not subscribed to the Entity:
image

## Testing

Preview link: https://deploy-preview-12--pablocosta-staging.netlify.app

### CODE REVIEW

**Comment by cesarvarela** [comment]
Preview link for this?

**Comment by pdcp1** [comment]
> Preview link for this?

@cesarvarela https://deploy-preview-12--pablocosta-staging.netlify.app

File: site/gatsby-site/cypress/e2e/entity.cy.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+};
+
+describe('Entities page', () => {
+  const url = `/entities/${entity.entity_id}`;
+
+  it('Successfully loads', () => {
+    cy.visit(url);
+  });
+
+  it('Should subscribe to new Entity incidents (authenticated user)', () => {
```
Comment (cesarvarela) [inline site/gatsby-site/cypress/e2e/entity.cy.js:15]: Tests that use the `login` method must use `maybeIt`

File: site/gatsby-site/cypress/e2e/entity.cy.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          upsertOneSubscription: {
+            _id: 'dummyIncidentId',
+          },
+        },
+      }
+    );
+
+    cy.contains(`Notify Me of New ${entity.name} Incidents`).scrollIntoView().click();
+
+    cy.wait(1000);
```
Comment (cesarvarela) [inline site/gatsby-site/cypress/e2e/entity.cy.js:40]: This will be flaky on CI 🤔

**Review by cesarvarela — CHANGES_REQUESTED** [review]
I think that this button should reflect notification status and allow you to toggle on/off:
image 🤔

File: site/gatsby-site/cypress/e2e/entity.cy.js (hunk 2 above)
Comment (pdcp1) [reply site/gatsby-site/cypress/e2e/entity.cy.js]: Removed

**Comment by pdcp1** [comment]
@cesarvarela I changed the button behavior to display the subscription status. Please review.
- If the user is subscribed to the Entity:
image

- if the user is not subscribed to the Entity:
image

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**